region/norfair/west/Ice Beam Snake Room.json

```diff
@@ -177,6 +179,44 @@
       "flashSuitChecked": true,
       "note": "It takes about 1.5 minutes for the Sova to get to this door and hit Samus."
     },
+    {
+      "link": [1, 2],
+      "name": "Come In Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 0,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementTricky",
+        {"or": [
+          {"enemyDamage": {
+            "enemy": "Sova",
+            "type": "contact",
+            "hits": 1
+          }},
+          "canInsaneJump"  
+        ]},
```

Comment:
I don't really know how to do this strat reliably. When I get an unmorph kago i'm usually too close to the edge and end up walking off instead of spin jumping. Jumping over seems even harder though: usually i either accidentally walk into the sova and/or activate the spark by straight jumping instead of spin jumping; even when getting the spin jump, about half the time i hit the sova on the way down before being able to spark.

Fitting into the hole usually worked for me but I think it must depend on the entry somehow. Moonfall or reset fall speed takes so many shinecharge frames, i'm always hitting the Sova before unmorphing in that case, and i don't see how it would be possible to get the spark off after that

Reply:
I think insaneJump fits always - I wasn't consistent with getting the shinespark off.
You could put `canOffScreenMovement` on it too.

And maybe check if there is time for a PB to help clear the Sovas.

Reply:
Ok, yeah a PB helps a lot. You still reach the Sovas before the explosion clears them, and there's not enough shinecharge frames to wait for it. But the 50% slowdown from the lag makes timing the unmorph and jump much easier. And the explosion can clear the Sova by the time you come down so you don't have to worry about hitting it then. It feels like it brings it down to Expert level. So I put `canInsaneJump` + `canOffScreenMovement` as an alternative to a PB, and removed the Sova damage requirement.

Added a new tech `canUsePowerBombLag` for this. I think it could be useful in many places. One reason to have this as a tech is in case the PB explosion lag could be patched out somehow (e.g. by overclocking, or with a PC port). I don't think we have interest in supporting PB lag removal as a Map Rando feature, but in case some other project does, it would be nice for it to be possible to account for it by disabling the tech. Come to think of it, this might affect players using the widescreen patch since I believe that requires overclocking; I'm not sure.

Reply:
Reducing an extreme strat to expert is where I see PowerBombLag fitting.  
ALTTPR has switched to fast ROM so it could carryover as a consideration for SMZ3.
It does help a lot here